### PR TITLE
Codenvy 694 remove unused field of WorkspaceManager and add getByNamespace manager mehod

### PR DESCRIPTION
**Pull request by mshaposhnik**

### CODE REVIEW

**Comment by garagatyi** [comment]
LGTM

File: wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java [hunk 1]
```diff
@@ -268,6 +264,27 @@ public WorkspaceImpl getWorkspace(String name, String namespace) throws NotFound
...
+     * @param namespace
+     *         the namespace to find workspaces
+     * @return the list of workspaces or empty list if no matches
+     * @throws NullPointerException
+     *         when {@code namespace} is null
+     * @throws ServerException
+     *         when any server error occurs while getting workspaces with {@link WorkspaceDao#getByNamespace(String)}
+     */
+    public List<WorkspaceImpl> getByNamespace(String namespace) throws ServerException {
+        requireNonNull(namespace, "Required non-null user namespace");
```
Comment (mshaposhnik) [inline wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java]: Word "user" not need here

**Comment by skabashnyuk** [comment]
ок, @sleshchenko ?

**Comment by sleshchenko** [comment]
Ok for me

**Pull request merged**